service/api/prettyprint.go

```diff
@@ -21,33 +21,48 @@ const (
 // SinglelineString returns a representation of v on a single line.
 func (v *Variable) SinglelineString() string {
 	var buf bytes.Buffer
-	v.writeTo(&buf, true, false, true, "", "")
+	v.writeTo(&buf, true, false, true, false, "", "")
+	return buf.String()
+}
+
+// SinglelineStringWithShortTypes returns a representation of v on a single line, with types shortened.
+func (v *Variable) SinglelineStringWithShortTypes() string {
```

Comment:
I decided to add a new function instead of adding a flag to `SinglelineString`, as that function has tons of call sites that I would have had to adapt.

Reply:
SinglelineStringFormatted as very few calls.

Reply:
Do you mean I should use `SinglelineStringFormatted` instead of adding `SinglelineStringWithShortTypes`? If so, should I change its behavior to always shorten types, or add a bool parameter to it to make it configurable? Personally I'd find it preferable to add the new function, as I don't need the ability to pass a format string.